sklearn/preprocessing/data.py

```diff
@@ -2002,15 +2002,15 @@ class CountFeaturizer(BaseEstimator, TransformerMixin):
     >>> cf = CountFeaturizer()
     >>> cf.fit_transform(data)
     array([[ 1.,  1.,  2.],
-    [ 1.,  1.,  2.],
-    [ 3.,  1.,  1.],
-    [ 0.,  0.,  1.]])
+           [ 1.,  1.,  2.],
+           [ 3.,  1.,  1.],
+           [ 0.,  0.,  1.]])
     >>> cf = CountFeaturizer(inclusion=[1])
     >>> cf.fit_transform(data)
     array([[ 1.,  1.,  3.],
-    [ 1.,  1.,  3.],
-    [ 3.,  1.,  3.],
-    [ 0.,  0.,  1.]])
+           [ 1.,  1.,  3.],
+           [ 3.,  1.,  3.],
+           [ 0.,  0.,  1.]])
```

Comment:
I am confused at this:

FAIL: Doctest: sklearn.preprocessing.data.CountFeaturizer

----------------------------------------------------------------------

Traceback (most recent call last):

  File "/usr/lib/python2.7/doctest.py", line 2201, in runTest

    raise self.failureException(self.format_failure(new.getvalue()))

AssertionError: Failed doctest test for sklearn.preprocessing.data.CountFeaturizer

  File "/home/travis/build/scikit-learn/scikit-learn/sklearn/preprocessing/data.py", line 1963, in CountFeaturizer

----------------------------------------------------------------------

File "/home/travis/build/scikit-learn/scikit-learn/sklearn/preprocessing/data.py", line 2003, in sklearn.preprocessing.data.CountFeaturizer

Failed example:

    cf.fit_transform(data)

Expected:

    array([[ 1.,  1.,  2.],

    [ 1.,  1.,  2.],

    [ 3.,  1.,  1.],

    [ 0.,  0.,  1.]])

Got:

    array([[ 1.,  1.,  2.],

           [ 1.,  1.,  2.],

           [ 3.,  1.,  1.],

           [ 0.,  0.,  1.]])

----------------------------------------------------------------------

File "/home/travis/build/scikit-learn/scikit-learn/sklearn/preprocessing/data.py", line 2009, in sklearn.preprocessing.data.CountFeaturizer

Failed example:

    cf.fit_transform(data)

Expected:

    array([[ 1.,  1.,  3.],

    [ 1.,  1.,  3.],

    [ 3.,  1.,  3.],

    [ 0.,  0.,  1.]])

Got:

    array([[ 1.,  1.,  3.],

           [ 1.,  1.,  3.],

           [ 3.,  1.,  3.],

           [ 0.,  0.,  1.]])

>>  raise self.failureException(self.format_failure(<StringIO.StringIO instance at 0x5fccb00>.getvalue()))

No matter what I do I can't get this test to pass. I have tried
 - Copying from terminal output
 - Copying from the docstring of other estimators/transformers 
 - Copying from the actual error output 
 - Trying to modify all sorts of space so that it would work as expected

Does anyone know what I should do to pass this docstring test?